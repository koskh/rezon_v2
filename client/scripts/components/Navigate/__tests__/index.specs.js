import * as React from 'react';
import {BrowserRouter as Router, Switch} from "react-router-dom";

import {shallow, mount} from 'enzyme';
import renderer from 'react-test-renderer';

import Navigate from '../index';

describe('Component/Navigate', () => {

    test('can render', () => {
        const component = renderer.create(<Router><Navigate /></Router>);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

});
