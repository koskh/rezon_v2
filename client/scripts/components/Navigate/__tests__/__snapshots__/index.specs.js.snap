// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/Navigate can render 1`] = `
<nav
  className="nav bg-light mb-4"
>
  <a
    aria-current={false}
    className="nav-link"
    href="/"
    onClick={[Function]}
    style={undefined}
  >
    Main
  </a>
  <a
    aria-current={false}
    className="nav-link"
    href="/tools"
    onClick={[Function]}
    style={undefined}
  >
    Tools
  </a>
  <a
    aria-current={false}
    className="nav-link"
    href="/login"
    onClick={[Function]}
    style={undefined}
  >
    Login
  </a>
  <a
    aria-current={false}
    className="nav-link"
    href="/about"
    onClick={[Function]}
    style={undefined}
  >
    About
  </a>
</nav>
`;
